cli/script/command-executor.ts

```diff
@@ -178,6 +178,24 @@ function appList(command: cli.IAppListCommand): Promise<void> {
         });
 }
 
+function checkIfAppExists(appName: string): Promise<App> {
+    log(chalk.cyan(`Checking if app with the name "${appName}" exists...\n`));
+    return Promise<App>((resolve, reject) => {
+        sdk.getApps()
```

Comment:
Oh, if you do this I guess you won't be able to return a list of candidate apps on failure, but note that none of the other commands return a list when they fail